provider-utils/awscloudformation/video-player-templates/video-player.ejs

```diff
@@ -139,4 +139,58 @@ export default Ember.Component.extend({
     });
   },
 });
-<% } %>
\ No newline at end of file
+<% } -%>
+<% if (framework === 'ios') { -%>
```

Comment:
Can we split the iOS/Android ones out of the JS one? Might just look cleaner

Reply:
Make sense! I added a `video-player.ejs` file in each directory (web/ios)